segmentstore/storage/impl/src/main/java/io/pravega/segmentstore/storage/impl/extendeds3/ExtendedS3StorageConfig.java

```diff
@@ -30,6 +30,7 @@
     public static final Property<String> URI = Property.named("url", "");
     public static final Property<String> BUCKET = Property.named("bucket", "");
     public static final Property<String> NAMESPACE = Property.named("namespace", ""); // use default namespace
+    public static final Property<Boolean> IFNONEMATCHWORKS = Property.named("ifnonematchworks", false);
```

Comment:
Let's rename this variable to something that resembles "use none match". In the format we are using for properties, are we allowed to use any separators, like "use.none.match"?

Rename the variable accordingly, please.